scripts/jsonnet/Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM golang:1.12-stretch
+FROM golang:1.12 as builder
 
 ENV JSONNET_VERSION 0.14.0
 ENV PROMTOOL_VERSION 2.12.0
 ENV GOLANGCILINT_VERSION 1.19.1
 
-RUN apt-get update -y && apt-get install -y g++ make git jq gawk python-yaml && \
+RUN apt-get update -y && apt-get install -y g++ make git && \
```

Comment:
@paulfantom  Just be sure, you are removing a couple of dependencies, I don't remember if I had tested those when I first introduced these changes, have you double-checked?

Reply:
I am removing them only from an intermediary builder image as those are not needed there, but we need them in a final one so I included those in https://github.com/coreos/prometheus-operator/pull/2846/files#diff-c2fbfe81c5dc36431cdcbb37e61e4d82R24

Reply:
And yes, I double checked while debugging https://github.com/prometheus/node_exporter/pulls?q=is%3Apr+sort%3Aupdated-desc+is%3Aclosed :)

Reply:
Ahh yes, I was them now